pkg/templates/templates.go

```diff
@@ -495,6 +505,23 @@ func (t *TemplateResolver) processForDataTypes(str string) string {
 	return processeddata
 }
 
+func (t *TemplateResolver) processCopiedDataTypes(str string) string {
+	d1 := regexp.QuoteMeta(t.config.StartDelim)
+	d2 := regexp.QuoteMeta(t.config.StopDelim)
+	re := regexp.MustCompile(
+		`:\s+(?:[\|>]-?\s+)?(?:'?\s*)(` + d1 + `.*(?:copyConfigMapData|copySecretData).*` + d2 + `)(?:\s*'?)`,
```

Comment:
Could you consolidate this method with `processForDataTypes`? This will allow us to execute the regex search once and also not copy the majority of the regex.

Reply:
I gave it a shot -- take a look.  Thanks